crates/pathfinder/src/sierra.rs

```diff
@@ -24,8 +24,9 @@ pub fn compile_to_casm(
         .parse_as_semver()
         .context("Deciding on compiler version")?
     {
-        Some(v) if v >= V_0_11_2 => v2::compile(definition),
-        Some(v) if v >= V_0_11_1 => v1_0_0_rc0::compile(definition),
+        Some(v) if v > V_0_11_2 => v2::compile(definition),
+        Some(v) if v == V_0_11_2 => v1_1_1::compile(definition),
+        Some(v) if v == V_0_11_1 => v1_0_0_rc0::compile(definition),
```

Comment:
Should we use the same convention or wdyt? Right now the `==` works but its more that each compiler just happens to cover a single version.
```suggestion
        Some(v) if v > V_0_11_2 => v2::compile(definition),
        Some(v) if v > V_0_11_1 => v1_1_1::compile(definition),
        Some(v) if v > V_0_11_0 => v1_0_0_rc0::compile(definition),
```

Reply:
Well, my way of thought was that `==` more explicitly states that it's covering a single version only.

It's maybe not that important though, I'm OK with `>` as well.